xml/art_modules.xml

```diff
@@ -692,7 +716,7 @@
     </listitem>
     <listitem>
      <para>
-      <emphasis role="bold">Lifecycle:</emphasis> 10 years
+      <emphasis role="bold">Lifecycle:</emphasis> December 2027 (Python 3.11)
```

Comment:
This is tricky, as the lifecycle of the SP4/SP5 module is the underlying SP lifecycle. it just happens to align here.